app/views/angular/index.html

```diff
@@ -0,0 +1,20 @@
+<html ng-app="loomioApp" ng-controller="ApplicationController as app">
```

Comment:
Beautiful, glorious, regular html, just like mom used to make.

(PS this is the mobile app entry point.)